### PR TITLE
chore: move system test utils to new library

### PR DESCRIPTION
**Pull request by lodmfjord**
# Create new library for system tests

## What

Foundation for moving system tests to their own project folders.

## Why

Currently @island-is/qa is the code owner of all system tests, which is not ideal.

### CODE REVIEW

**Review by AndesKrrrrrrrrrrr — APPROVED** [review]

File: libs/testing-system/src/index.ts
Comment (saevarma) [inline libs/testing-system/src/index.ts:1]: Please export here directly and remove the `testing-system.ts` file.

**Review by saevarma — CHANGES_REQUESTED** [review]
I assume that this PR is moving files from `apps/system-e2e/src/support|utils` to this new library. This PR should also remove those file from `apps/system-e2e` and import from this new library.

**Comment by andes-it** [comment]
This PR's had a 15-day siesta. Lost its spark, did it? It's making an exit in 5 days unless the stale label disappears.

**Comment by andes-it** [comment]
This PR closed its doors. Sat around for a day after the stale tag, then reckoned it's not worth the fuss.

**Pull request closed without merging**